src/Symfony/Component/Validator/Tests/Constraints/NotJsonValidatorTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\NotJson;
+use Symfony\Component\Validator\Constraints\NotJsonValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+class NotJsonValidatorTest extends ConstraintValidatorTestCase
+{
+    protected function createValidator()
+    {
+        return new NotJsonValidator();
+    }
+
+    public function testJsonIsValid()
+    {
+        $this->validator->validate('{ "planet":"earth", "country": "Morocco", "city": "Rabat" }', new NotJson());
```

Comment:
perhaps add some cases for primitive values (null, ints, bools, strings)

Reply:
I've added others cases with primitive values (y)